x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/alert_details_app_section/latency_alerts_history_chart.tsx

```diff
@@ -69,7 +71,7 @@ export function LatencyAlertsHistoryChart({
     end,
     kuery: '',
     numBuckets: 100,
-    type: ApmDocumentType.ServiceTransactionMetric,
+    type: ApmDocumentType.TransactionMetric,
```

Comment:
@dgieselaar @sorenlouv @benakansara Any idea what is this type?

After changing the type, the issue mentioned here is solved but I don't know why and whether it can have a negative impact on other cases when trnasactionName is not included in the group fields. (I will test this feature later but thought maybe you already know the answer.)

Reply:
@maryam-saeidi service transaction metrics do not have transactionName as a dimension - so if transactionName is set, `ApmDocumentType.TransactionMetric` should be used instead of `ApmDocumentType.ServiceTransactionMetric`. In other cases, you'll want to use the latter, as it's (a lot) faster.